src/windows/title_editor.py

```diff
@@ -426,83 +442,45 @@ def btnFont_clicked(self):
             self.font_weight = fontinfo.weight()
             self.font_size_pixel = fontinfo.pixelSize()
             self.set_font_style()
-
-            # Something changed, so update temp SVG
-            self.writeToFile(self.xmldoc)
-
-            # Display SVG again
-            self.display_svg()
+            self.save_and_reload()
 
     def find_in_list(self, l, value):
         '''when passed a partial value, function will return the list index'''
-        for item in l:
+        for i, item in enumerate(l):
             if item.startswith(value):
-                return l.index(item)
+                return i
```

Comment:
I also goosed the efficiency of `find_in_list()` a bit, since it's still used in a couple of parts of the code.